lib/suspenders/generators/app_generator.rb

```diff
@@ -4,7 +4,7 @@
 module Suspenders
   class AppGenerator < Rails::Generators::AppGenerator
     class_option :database, type: :string, aliases: "-d", default: "postgresql",
-      desc: "Configure for selected database (options: #{DATABASES.join("/")})"
+      desc: "Configure for selected database (options: #{DATABASES.push('mongodb').join("/")})"
```

Comment:
Line is too long. [95/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.